bookkeeper-server/src/main/java/org/apache/bookkeeper/client/RoundRobinDistributionSchedule.java

```diff
@@ -62,10 +62,11 @@ public RoundRobinDistributionSchedule(int writeQuorumSize, int ackQuorumSize, in
 
     @Override
     public void getWriteSet(long entryId, int[] target) {
-        if (target.length != writeQuorumSize) {
```

Comment:
any reason why you can this behavior? isn' t `!=` a stricter validation?

Reply:
Because as PendingAddOps get reused, a PendingAddOp with a writeSet of size 5 may get used with a ledger that needs a writeSet of size 3.